lib/elixir/test/elixir/calendar/holocene.exs

```diff
@@ -154,4 +154,21 @@ defmodule Calendar.Holocene do
 
   @impl true
   defdelegate iso_days_to_end_of_day(iso_days), to: Calendar.ISO
+
+  # The Holocene calendar extends most year and day count guards implemented in the ISO calendar,
+  # therefore we just return the input date parts.
+  @impl true
+  def shift_date(year, month, day, _duration) do
+    {year, month, day}
```

Comment:
Maybe it is best to just raise? And we can ensure the calendar is invoked by rescueing the raise? This way nobody will call it by "accident".